modules/openideal_idea/config/optional/core.entity_view_display.node.idea.default.yml

```diff
@@ -327,6 +329,20 @@ content:
     third_party_settings: {  }
     type: entity_reference_label
     region: content
+  field_five_stars:
```

Comment:
Is it expected to have one five-star field by default?

Reply:
Yeah.